docs/v0.5.x/lightning-strategy-examples.md

```diff
@@ -40,11 +40,13 @@ end
 
 The nice thing about this is that you can deploy your app to production, test it out by passing an index_key parameter with the revision you want to test and activate when you feel confident that everything is working as expected.
 
-## Example Node app (outdated)
+## Example Node app
 
-This app does the same as the *Sinatra app* above, it supports the same index_key query param. It should help you to get up and running in seconds and dont worry about server code. You also need to deploy your ember app with `ember-cli-deploy` and `ember-deploy-redis`.
```

Comment:
ok this mention of `ember-deploy-redis` is wrong.

maybe keep your description, rename the section to `example node apps` and leave at the end the link to ember-lightning?